examples/webgl_buffergeometry_instancing_lambert.html

```diff
@@ -297,71 +220,20 @@
 			geometry.addAttribute( 'instanceColor', new THREE.InstancedBufferAttribute( colors, 3 ) );
 			geometry.addAttribute( 'instanceScale', new THREE.InstancedBufferAttribute( scales, 1 ) );
 
-
 			// material
 
 			var envMap = new THREE.TextureLoader().load( `textures/metal.jpg`, function ( texture ) {
 
 				texture.mapping = THREE.SphericalReflectionMapping;
-
+				material.needsUpdate = true
 			} );
 
-			var shader = THREE.CustomShaderLib[ 'lambert' ];
-
-			var uniforms = THREE.UniformsUtils.clone( shader.uniforms );
-
-			uniforms[ "diffuse" ].value.set( 0xffb54a );
```

Comment:
This is too different from the `MeshLambertMaterial`'s interface `material.color`